lib/src/components/NxBackButton/__tests__/NxBackButton.test.tsx

```diff
@@ -4,37 +4,47 @@
  * the terms of the Eclipse Public License 2.0 which accompanies this
  * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
  */
-import * as enzymeUtils from '../../../__testutils__/enzymeUtils';
-import 'jest-enzyme';
+
+import { rtlRender, rtlRenderElement } from '../../../__testutils__/rtlUtils';
+import { within } from '@testing-library/react';
+
 import NxBackButton, { Props } from '../NxBackButton';
 
 describe('NxBackButton', function() {
-  const getShallowComponent = enzymeUtils.getShallowComponent<Props>(NxBackButton, { href: '/foo' });
+  const quickRender = rtlRender<Props>(NxBackButton, { href: '/foo' }),
```

Comment:
Separate out the `minimalProps` into a separate variable

Reply:
> Separate out the `minimalProps` into a separate variable

added here: [62efa71](https://github.com/sonatype/sonatype-react-shared-components/pull/914/commits/62efa713e5c053127f29012f250b5cc1315484b2)